service/expression.js

```diff
@@ -1,64 +1,30 @@
-import { User, Expression, UserExpression } from '../models';
+import {
+  getExpressionByValue,
+  addExpression,
+  addExpressionToUser,
+  getExpressionsByUser,
+  deleteExpression,
+  deleteExpressionFromUsers,
+} from '../repository/expression';
 
 export const insertExpression = async (userId, value) => {
-  try {
-    const expressionFound = await Expression.findOne({
-      where: {
-        value,
-      },
-    });
+  const expressionFound = await getExpressionByValue(value);
 
-    const { id } = expressionFound
-      ? expressionFound
-      : await Expression.create({ value });
+  const { id } = expressionFound ? expressionFound : await addExpression(value);
 
-    return await UserExpression.create({
-      userId,
-      expressionId: id,
-    });
-  } catch (error) {
-    throw new Error(error.message);
-  }
+  return await addExpressionToUser(userId, id);
```

Comment:
`return await` is somehow redundant. You pause the execution here to wait for the promise to finish but you dont use the value here. You should await the promise right before you use the value.